tests/NuGet.Services.BasicSearchTests/Data/V2SearchFunctionalTests.ReturnsCorrectSchema.json

```diff
@@ -1,68 +1,64 @@
 ﻿{
-    "totalHits": 2,
-    "index": "RAM",
-    "indexTimestamp": "1/1/0001 8:00:00 AM",
-    "data": [
-        {
-            "PackageRegistration": {
-                "Id": "EntityFramework",
-                "DownloadCount": 50,
-                "Verified": false,
-                "Owners": []
-            },
-            "Version": "6.1.3-beta1",
-            "NormalizedVersion": "6.1.3-beta1",
-            "Title": "EntityFramework",
-            "Description": "Entity Framework is Microsoft's recommended data access technology for new applications.",
-            "Summary": "Entity Framework is Microsoft's recommended data access technology for new applications.",
-            "Authors": "Microsoft",
-            "Language": "en-US",
-            "Tags": "Microsoft EF Database Data O/RM ADO.NET",
-            "ProjectUrl": "http://go.microsoft.com/fwlink/?LinkID=320540",
-            "IconUrl": "http://go.microsoft.com/fwlink/?LinkID=386613",
-            "IsLatestStable": false,
-            "IsLatest": true,
-            "Listed": true,
-            "Published": "1/1/0001 8:00:00 AM",
-            "LastUpdated": "1/1/0001 8:00:00 AM",
-            "DownloadCount": 30,
-            "Dependencies": [],
-            "SupportedFrameworks": [],
-            "Hash": "e7xbxwBo7NKNx1ZeyVgAdYdjl2diR71A4koclGbgQfQyV+8/7AhXpuiOF9mOHkg1f6M7aRVMbMhwSS+TRJHvWg==",
-            "HashAlgorithm": "SHA512",
-            "PackageFileSize": 4255228,
-            "LicenseUrl": "http://go.microsoft.com/fwlink/?LinkID=320539",
-            "RequiresLicenseAcceptance": false
-        },
-        {
-            "PackageRegistration": {
-                "Id": "angularjs",
-                "DownloadCount": 10,
-                "Verified": true,
-                "Owners": []
-            },
-            "Version": "1.4.8",
-            "NormalizedVersion": "1.4.8",
-            "Title": "AngularJS",
-            "Description": "AngularJS. HTML enhanced for web apps!",
-            "Summary": "",
-            "Authors": "Fitzchak Yitzchaki,Dov Landau",
-            "Copyright": "",
-            "Tags": "angular angularjs SPA",
-            "ProjectUrl": "http://angularjs.org",
-            "IconUrl": "https://secure.gravatar.com/avatar/6e1b5ab3ef1593413f1bee4e5a6e6ae7?s=140&d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-140.png",
-            "IsLatestStable": true,
-            "IsLatest": true,
-            "Listed": true,
-            "Published": "1/1/0001 8:00:00 AM",
-            "LastUpdated": "1/1/0001 8:00:00 AM",
-            "DownloadCount": 10,
-            "Dependencies": [],
-            "SupportedFrameworks": [],
-            "Hash": "wX6vTdbH2OyLRcRLxIqYUXc4pNRAk6aRtvduu2PFggpP+UcKj4hznol1RAFnm9DIRr5SgxlkOYJacTZyHYMcpg==",
-            "HashAlgorithm": "SHA512",
-            "PackageFileSize": 1838700,
-            "RequiresLicenseAcceptance": false
-        }
-    ]
-}
+  "totalHits": 2,
```

Comment:
This file diff is mostly whitespace change.  To view the file diff without whitespace changes go to https://github.com/NuGet/NuGet.Services.Metadata/pull/352/files?w=1 .